rundeckapp/grails-spa/packages/ui-trellis/src/app/components/server-identity/serverIdentity.ts

```diff
@@ -0,0 +1,55 @@
+// The Vue build version to load with the `import` command
+// (runtime-only or standalone) has been set in webpack.base.conf with an alias.
+import {defineComponent, markRaw} from 'vue'
+import ServerDisplay from '../../../library/components/version/ServerDisplay.vue'
+import {getRundeckContext} from '../../../library'
+import {ServerInfo} from '../../../library/stores/System'
+
+const ServerIdentityComp = defineComponent({
+  name: 'ServerIdentityComp',
+  components: {
+    ServerDisplay
+  },
+  data() {
+    return {
+      serverInfo: null
+    }
+  },
+  props: {
+    showId: {type: Boolean, default: true},
+    nameClass: {type: String, default: ''},
+    serverName: {type: String, default: ''},
+    serverUuid: {type: String, default: ''}
+  },
+  async mounted() {
+      this.serverInfo = new ServerInfo(this.serverName, this.serverUuid)
+  },
+  template: `
+    <server-display :uuid="serverInfo.uuid"
+                    :name="serverInfo.name"
+                    :name-class="nameClass"
+                    :glyphicon="serverInfo.icon"
+                    :show-id="showId"
+                    v-if="serverInfo">
+    </server-display>`
+})
+
+let rundeckContext = getRundeckContext()
```

Comment:
As this variable is not going to be reassigned is better to set it as `const`
```suggestion
const rundeckContext = getRundeckContext()
```